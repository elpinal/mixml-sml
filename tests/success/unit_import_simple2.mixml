[unit
  [unit :{type a : Type} import (a)]
]
