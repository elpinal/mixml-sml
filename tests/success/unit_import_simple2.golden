[[forall (!3@a : Type). {a : [= !3]}]-]+
